rbc/heavydb/column_list.py

```diff
@@ -91,6 +91,7 @@ def codegen(context, builder, signature, args):
 class ColumnList(Buffer):
     """
     RBC ``ColumnList<T>`` type that corresponds to HeavyDB COLUMN LIST
+    of arrays
```

Comment:
hmm, why arrays? It could be any of any support types.

Reply:
perhaps this was meant for ColumnListArray doc-string..

Reply:
Yes, ColumnListArray